easybuild/easyconfigs/l/libevent/libevent-2.1.8.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'libevent'
+version = '2.1.8'
+
+homepage = 'http://libevent.org/'
+description = """The libevent API provides a mechanism to execute a callback function when a specific
+ event occurs on a file descriptor or after a timeout has been reached.
+ Furthermore, libevent also support callbacks due to signals or regular timeouts."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = [
+    'https://github.com/%(name)s/%(name)s/releases/download/release-%(version)s-stable/',
+    'https://downloads.sourceforge.net/project/levent/release-%(version)s-stable/',
+]
+
+sources = ['%(name)s-%(version)s-stable.tar.gz']
+
+checksums = ['f3eeaed018542963b7d2416ef1135ecc']
+
+sanity_check_paths = {
+    'files': ["bin/event_rpcgen.py"] +
+             ['include/%s' % x for x in ["evdns.h", "event.h", "evhttp.h", "evrpc.h", "evutil.h"]] +
+             ['include/event2/%s' % x for x in ['buffer_compat.h', 'bufferevent.h', 'bufferevent_struct.h', 'dns_compat.h', 'dns_struct.h', 'event-config.h', 'event_struct.h', 'http.h', 'keyvalq_struct.h', 'rpc_compat.h', 'rpc_struct.h', 'tag.h', 'util.h', 'bufferevent_compat.h', 'bufferevent_ssl.h', 'buffer.h', 'dns.h', 'event_compat.h', 'event.h', 'http_compat.h', 'http_struct.h', 'listener.h', 'rpc.h', 'tag_compat.h', 'thread.h', 'visibility.h']],
```

Comment:
@foxik0070 please avoid using lines longer than 120 characters

In this case, I would move the check for `include/event2` to `dirs` rather than listing all header files?